rfcs/text/Fusillade in the DCP.md

```diff
@@ -0,0 +1,105 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Fusillade Management Processes
+
+*Replace "RFC Name" above with the name of your RFC. Keep it short and descriptive.*
+
+## Summary
+
+Define a series of process for managing fusillade across the components of the DCP.
+
+## Author(s)
+
+*Recommended format for Authors:*
+
+ `[Trent Smitg](mailto:trent.smith@chanzuckerberg.com)`
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Fusillade is responsible for providing a uniform authentication and authorization experiance across the DCP. As the DCP 
+moves forward with integrating fusillade into the DCP, processes needs to be put into place to prevent excessive 
+breaking of the system due to permission errors. By agreeing on a process for managing the permission in Fusillade, 
+errors relating to permissioning can be avoided, and will be easier to track down when they occur. 
+
+The goal of have an agreed process is to simplify the manage of Fusillade Roles, Groups, Users, Service Accounts across 
+the DCP.
+ 
+[fusillade:#226](https://github.com/HumanCellAtlas/fusillade/issues/226)
+
+### Glossary
+
+### User Stories
+
+1. As a component of the DCP, I would like to define permissions to limit access to specific resources.
+1. As a component of the DCP, I would like to define a group of users to manage the permission of a large groups of 
+users easily.
+1. As a component of the DCP, I would like to give permission to unauthenticated users.
+1. As a component of the DCP, I would like to use service account to programmatically manage my service.  
+
+## Scientific "guardrails" [optional]
+
+*Describe recommended or mandated review from HCA Science governance to ensure that the RFC addresses the needs of the scientific community.*
+
+## Detailed Design
+
+### As a component of the DCP, how do I control access to my resources?
+1. Define what actions and resources you'd like to control. Actions should take the form `{component}:{Action}`, for example to get a bundle from the DSS the action could be `dss:GetBundle`. The action is case sensitive, and camel case is suggested. Resources are what actions affect. The same action can be applied to multiple different resources. For the resource use the AWS ARN format. For example...
```

Comment:
Should there be a distinction between `dss:GetBundle` for AWS and GCP?

Reply:
My opinion is that this would be a field in the resource.

Reply:
Components have the freedom to determine the relationship between actions and resources. I will expand on the guide lines for defining resources, and add some examples.

Reply:
I added additional documentation for resources to help make this clear for components